phpunit/block-supports/block-style-variations-test.php

```diff
@@ -63,6 +63,11 @@ public function filter_set_theme_root() {
 	public function test_add_registered_block_styles_to_theme_data() {
 		switch_theme( 'block-theme' );
 
+		// Trigger block style registration that occurs on `init` action.
+		// do_action( 'init' ) could be used here however this direct call
+		// means only the updates being tested are performed.
+		gutenberg_register_block_style_variations_from_theme();
```

Comment:
Is it better to call the registration function direct here or call `do_action( 'init' );`?